docs/graphite.md

```diff
@@ -47,7 +47,9 @@ Here are the currently included functions:
 | movingAverage(seriesLists, windowSize) seriesList     |             | Unstable   |
 | perSecond(seriesLists) seriesList                     |             | Stable     |
 | rangeOfSeries(seriesList) series                      |             | Stable     |
-| scale(seriesList, num) series                         |             | Stable     |
+| removeAbovePercentile(seriesList, n) seriesList       |             | Stable     |
+| removeBelowPercentile(seriesList, n) seriesList       |             | Stable     |
+| scale(seriesLists, num) series                        |             | Stable     |
```

Comment:
why the change to scale? graphite's scale only takes a single serieslist. do we allow multiple? (it's possible, would need to dig into the code)

Reply:
Sorry this is merge weirdness. It used to say `seriesLists` incorrectly, and I fixed it in one of my commits, but when I rebased this branch, it pulled the old value.

Reply:
ok